tests/test_exact_dedup.py

```diff
@@ -24,7 +24,7 @@
 
 
 @pytest.fixture(params=[False, pytest.param(True, marks=pytest.mark.gpu)], ids=["no gpu", "gpu"])
-def exact_dedup_data(request: pytest.SubRequest) -> DocumentDataset:
+def exact_dedup_data(request) -> DocumentDataset:  # noqa: ANN001
```

Comment:
In my tests I figured the datatype of this is `from pytest import FixtureRequest`